root/edit/details/AddStandaloneRecording.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+import formatTrackLength from
+  '../../static/scripts/common/utility/formatTrackLength';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AddStandaloneRecordingProps = {
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string | null,
+    +length: number | null,
+    +name: string,
+    +recording: RecordingT,
+    +video: boolean,
+  },
+};
+
+const AddStandaloneRecording = (
+  {edit}: {+edit: AddStandaloneRecordingProps},
+) => {
+  const display = edit.display_data;
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Recording'))}</th>
+            <td><EntityLink allowNew entity={display.recording} /></td>
```

Comment:
I know this is what we use now, but maybe we should change to DescriptiveLink? That would show the artist credit here. Admittedly, the artist is shown further down, but that's the original artist credit for the edit, while this would be the current one :) @mwiencek, @yvanzo: what do you think?

Reply:
Is there any reason to display current data along with edit data to start with? There is nothing that tells users `Recroding` is not edit data, it is quite confusing. If it is neither needed to display current edits nor needed to display past edits, I would just drop the whole `Recording` row.

Reply:
@mwiencek Whats your take on this ?

Reply:
@yvanzo: Well, it helps me figure out if it was changed again since this edit, so I personally find it useful :) But I'm not sure if that's just me or a more general situation.

Reply:
I could see cases in which it might be useful, but I'd move it outside from under the `Changes` header (maybe above it), since it's not a change.

Reply:
Since that's actually outside of the edit template, maybe just leave it be for now.